lib/compiler.js

```diff
@@ -12,6 +12,11 @@ const chalk = require('chalk');
 const webpack = require('webpack');
 
 const run = ({ config, emitJson, watchConfig }, log) => {
+  // Delete possible watch flag from webpack configuration as that triggers a
+  // faulty condition in webpack 5. #20
+  // eslint-disable-next-line no-param-reassign
+  delete config.watch;
```

Comment:
wouldn't this need to be only for >= v4?

Reply:
I added a check. I also changed it so that we don't do the duplicate hash check for webpack 5 as the watch mechanism is different there (duplicate is expected?).